org.robomongo.Robo3T.json

```diff
@@ -0,0 +1,51 @@
+{
+    "app-id": "org.robomongo.Robo3T",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "robo3t",
+    "finish-args": [
+        "--socket=x11",
```

Comment:
you need to add `--share=ipc` as well if you're using x11, it significantly improves performance. http://docs.flatpak.org/en/latest/sandbox-permissions-reference.html#f1